docs/configuration.md

```diff
@@ -44,6 +44,7 @@ Every variable is optional, though you should enable at least 1 backend.
   * The items page.
   * The graph page's tag list.
 * graphiteHost: an ip, hostname, ip:port, hostname:port or a URL, defaults to standard http/https ports, defaults to "/render" path.  Any non-zero path (even "/" overrides path)
+* graphiteHeader: a http header to be sent to graphite on each request in 'key:value' format. optional. can be specified multiple times.
```

Comment:
you're right, fixed in https://github.com/bosun-monitor/bosun/pull/1234
thanks for still looking out matt ;)